freedomofpress/pressfreedomtracker.us
# Use chart or visualization as primary image

closes #1658

Allows `IncidentsTimeBarChart`, `IncidentsTreeMap`, or `IncidentsBubbleMap` to be used as a primary image on blog posts, showing up at the top of the post. The chart in this position is static and cannot be interacted with.

## Screenshots of charts as primary image on blog posts

![Screenshot 2023-06-13 at 6 43 12 PM](https://github.com/freedomofpress/pressfreedomtracker.us/assets/3477162/8ccdb026-0540-4f83-ac92-0ba7d2a8a71d)
![Screenshot 2023-06-13 at 6 43 45 PM](https://github.com/freedomofpress/pressfreedomtracker.us/assets/3477162/61c3a72b-330b-4c77-8e3a-387faeb40149)
![Screenshot 2023-06-06 at 3 59 46 PM](https://github.com/freedomofpress/pressfreedomtracker.us/assets/3477162/9cbab4ed-e5cb-4f19-a5ee-47fb78d8876a)
![Screenshot 2023-06-13 at 6 42 46 PM](https://github.com/freedomofpress/pressfreedomtracker.us/assets/3477162/a54928c0-f345-46ce-ab4b-a305363c61d4)

## Screenshot of charts as teaser image on blog posts listing pages

![Screenshot 2023-06-14 at 5 38 57 PM](https://github.com/freedomofpress/pressfreedomtracker.us/assets/3477162/3532343b-8b81-4968-b6cc-cc71277a0817)

## To test and verify
- In wagtail, set the `lead_graphic` to a chart and it should show up in the primary image position of the blog post.
- In wagtail, set the `teaser_graphic` to a chart and it should show up as the teaser image position on the blog post listing pages.
